README.rst

```diff
@@ -37,6 +37,7 @@ numpy            1.9.0
 nibabel          1.4.0dev
 pydicom          0.9.9
 dcmstack         0.7.0dev
+pymongo          2.8
```

Comment:
I could just rebase our commits to latest master and get rid of this instead...?

Do you know if the latest master should work for me, or do I still need to be on a special branch (new_deploy I think it's called)?